code/modules/organs/internal/lungs.dm

```diff
@@ -80,6 +80,8 @@
 		poison_types =        species.poison_types || list(/decl/material/gas/chlorine = TRUE)
 		exhale_type =         species.exhale_type  || /decl/material/gas/carbon_dioxide
 	else
+		//Species-less lungs should be illegal
+		log_debug("Got null species")
```

Comment:
Might want to make this debug message more specific

Reply:
Yeah, that shouldn't be in here. I was debugging why mobs would runtime when breathing.